packages/node_modules/@ciscospark/widget-space/src/container.js

```diff
@@ -102,6 +114,8 @@ export class SpaceWidget extends Component {
       );
     }
     if (sparkInstance && spaceDetails) {
+      // Filter out activities that are disabled using the spaceActivities prop
+      activityTypes = activityTypes.filter((a) => spaceActivities[a.name] !== false);
```

Comment:
Sorry, just noticed this. This filter should probably be placed in a selector so that it doesn't run every time the widget renders.

Reply:
Will do!